test/private/privateAPIs.spec.ts

```diff
@@ -323,6 +324,26 @@ describe('MicrosoftTeams-privateAPIs', () => {
     expect(message.args[11]).toBe('view');
   });
 
+  it('should treat messages to frameless windows as coming from the child', () => {
+    const parent = utils.initializeAsFrameless(null, ['https://www.example.com']);
+
+    // Simulate recieving a child message as a frameless window
+    utils.processMessage({
+      origin: 'https://www.example.com',
+      source: utils.childWindow,
+      data: {
+        id: 0,
+        func: 'themeChange',
+        args: ['testTheme'],
+      } as MessageResponse,
+    } as MessageEvent);
+
+    // The frameless window should send a response back to the child window
+    expect(utils.childMessages.length).toBe(1);
+
+    utils.mockWindow.parent = parent;
```

Comment:
I understand why this is necessary but it's a little confusing for this cleanup code to live here. Can we move this into either the `initializeWithContext` helper or at least into a `beforeEach` function that resets this before every test case?

Reply:
Done 👍